protos/sigstore_rekor.proto

```diff
@@ -128,3 +128,12 @@ message TransparencyLogEntry {
         // payload from other sources to verify the signature.
         bytes canonicalized_body = 7;
 }
+
+// The RekorBundle is the signed material used to produce the Signed Entry
+// Timestamp signature. See notes on the InclusionPromise above.
+message RekorBundle {
```

Comment:
naming nit: I don't want to proliferate "bundle" even though it's in cosign, given we want "bundle" to refer to top level protobuf bundle now. Do you have another suggestion of the name? SignedEntryTimestampContents?